dist/tooltip/ds6/tooltip.css

```diff
@@ -198,9 +252,15 @@
   width: 16px;
   top: calc(50% - 8px);
 }
-.tooltip-comp .tooltip__arrow-right-bottom,
-.tooltip-comp .infotip__arrow-right-bottom,
-.tooltip-comp .tourtip__arrow-right-bottom {
+.tooltip--tooltip .tooltip__arrow-right-bottom,
+.tooltip--infotip .tooltip__arrow-right-bottom,
+.tooltip--tourtip .tooltip__arrow-right-bottom,
+.tooltip--tooltip .infotip__arrow-right-bottom,
+.tooltip--infotip .infotip__arrow-right-bottom,
+.tooltip--tourtip .infotip__arrow-right-bottom,
+.tooltip--tooltip .tourtip__arrow-right-bottom,
+.tooltip--infotip .tourtip__arrow-right-bottom,
+.tooltip--tourtip .tourtip__arrow-right-bottom {
```

Comment:
This feels like a little excess duplication. We can probably get away with simply:
```
.tooltip .arrow-right-bottom
```
Then we can simplify the HTML, too by always referring to the arrow positions no matter the variation we're talking about. It's a much better use of the CSS cascade, too.

Thoughts?

Reply:
I agree.  In the specs the arrow position doesn't change based on the type of tooltip.  So no need to duplicate the arrow styles for each type.